api/src/main/java/org/pmiops/workbench/monitoring/MonitoringService.java

```diff
@@ -1,20 +1,51 @@
 package org.pmiops.workbench.monitoring;
 
+import com.google.common.collect.ImmutableMap;
+import io.opencensus.metrics.data.AttachmentValue;
+import java.util.Collection;
+import java.util.Collections;
 import java.util.Map;
-import org.pmiops.workbench.monitoring.views.OpenCensusStatsViewInfo;
+import org.pmiops.workbench.monitoring.views.EventMetric;
+import org.pmiops.workbench.monitoring.views.OpenCensusView;
 
 public interface MonitoringService {
 
+  int DELTA_VALUE = 1;
+  Map<String, AttachmentValue> DEFAULT_ATTACHMENTS = Collections.emptyMap();
+
   /**
-   * Record an occurrence of a counted event.
+   * Record an occurrence of a counted (a.k.a. delta or cumulative) time series. These are events
+   * that are typically measured more for frequency than for absolute value.
    *
-   * @param viewInfo
+   * @param eventMetric
    */
-  default void recordIncrement(OpenCensusStatsViewInfo viewInfo) {
-    recordValue(viewInfo, 1);
+  default void recordEvent(EventMetric eventMetric) {
```

Comment:
I almost named all of these `record` and let the overloads do their thing, but I thought it would be clearer to be opinionated. E.g. you should never record a gauge metric without a value, or an event metric with a value.